src/poetry/vcs/git/backend.py

```diff
@@ -329,7 +330,7 @@ def _clone_submodules(cls, repo: Repo) -> None:
         modules_config = repo_root.joinpath(".gitmodules")
 
         if modules_config.exists():
-            config = ConfigFile.from_path(modules_config)
+            config = ConfigFile.from_path(modules_config.as_posix())
```

Comment:
Not sure if this is safe (on e.g. Windows) but I see this is how `Path` objects are stringified all over the code base.

Reply:
`str(path)`